### PR TITLE
oem-ibm: Capacity on demand licensing support

### PR DESCRIPTION
**Pull request by jaypadath**
This commit supports:

1. Creation or update of new license interfaces and properties supplied by host
2. Clearing of license status to Unknown, during host power off transition
3. Restoration of license interfaces during reset reload and setting the license status to Unknown

Tested By:

1. Verified that dbus hosting is done by pldm
2. CoD(Capacity on demand) values are coming up in the BMC
3. Verified that reboot of BMC does not clear values

Change-Id: I23e7e6a533ef869329abe555790b584430f3f8c4

### CODE REVIEW

**Review by Pavithrab7 — APPROVED** [review]
Jay, changes look good.

File: oem/ibm/libpldmresponder/file_io_type_cert.hpp [hunk 1]
```diff
@@ -51,7 +51,7 @@ class CertHandler : public FileHandler
 
     virtual int newFileAvailable(uint64_t length);
 
-    virtual int fileAckWithMetaData(uint8_t /*fileStatus*/,
+    virtual int fileAckWithMetaData(uint8_t fileStatus,
```
Comment (Patel-Kamalkumar) [inline oem/ibm/libpldmresponder/file_io_type_cert.hpp:54]: why this change? if change is required then cert.cpp file is not present for review

**Review by Patel-Kamalkumar — APPROVED** [review]
changes looks good to me

**Comment by jaypadath** [comment]
jenkins run tests please
